# Fix needed for Firefox
# Need to determine if these changes are still applicable to the latest version,
# and if so, then submit them upstream. 
--- sqlite-autoconf-3090200/Makefile.am.orig	2015-12-21 15:29:12.245364514 +0530
+++ sqlite-autoconf-3090200/Makefile.am	2015-12-21 15:29:36.374953130 +0530
@@ -3,7 +3,7 @@ AM_CFLAGS = @THREADSAFE_FLAGS@ @DYNAMIC_
 
 lib_LTLIBRARIES = libsqlite3.la
 libsqlite3_la_SOURCES = sqlite3.c
-libsqlite3_la_LDFLAGS = -no-undefined -version-info 8:6:8
+libsqlite3_la_LDFLAGS = -no-undefined -version-info 8:6:8 -Wl,-M,mapfile-libsqlite3
 
 bin_PROGRAMS = sqlite3
 sqlite3_SOURCES = shell.c sqlite3.c sqlite3.h
